src/tests/Interop/COM/NETClients/Primitives/TestInALC.cs

```diff
@@ -10,7 +10,7 @@ class Test
     {
         static int Main(string[] args)
         {
-            string currentAssemblyDirectory = Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().CodeBase).AbsolutePath);
+            string currentAssemblyDirectory = Path.GetDirectoryName(new Uri(Assembly.GetExecutingAssembly().Location).AbsolutePath);
```

Comment:
```suggestion
            string currentAssemblyDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```
`AssemblyCodeBase` was URL. `Assembly.Location` is file path.

Reply:
Thanks. I verified that wrapping each in `new Uri` got the same net result, but then I failed to catch that the `Uri` was an inconvenient detail here.

Reply:
Multiple files in this PR have the same issue.

Reply:
Thanks; I'll fix 'em all